src/relay/mod.rs

```diff
@@ -1,7 +1,8 @@
 //! Relay server in local and server side implementations.
 
-pub mod dns;
+// pub mod dns;
```

Comment:
This `dns` mod seems unused. Is it legacy module that is no longer used?

Reply:
There is a binary `ssdns`, which have been removed long time ago.

Reply:
Thanks! Ready to review now.